oomact_ros/src/CalibrationNode.cpp

```diff
@@ -5,98 +5,113 @@
 #include <aslam/calibration/ros/RosInputProvider.h>
 #include <aslam/calibration/calibrator/CalibratorI.hpp>
 #include <aslam/calibration/model/sensors/PoseSensor.hpp>
+#include <aslam/calibration/model/sensors/PositionSensor.hpp>
+
+#include <sm/BoostPropertyTree.hpp>
+#include <sm/value_store/PropertyTreeValueStore.hpp>
 
 namespace cal = aslam::calibration;
 
-int main(int argc, char **argv) {
+std::string getFilePathFromRosParam(const std::string& param_name,
+                                    ros::NodeHandle* nh) {
+  std::string file_path;
+  if (!nh->getParam(param_name, file_path)) {
+    ROS_FATAL_STREAM("Could not find " << file_path << ", exiting");
+    exit(EXIT_FAILURE);
+  }
+  return file_path;
+}
+
+void loadSensorParameters(const cal::ValueStoreRef vs_sensors,
```

Comment:
At some point this functionality (create objects from config) should go to the core library :).